avaframe/out3Plot/outAIMEC.py

```diff
@@ -162,13 +222,60 @@ def visuRunout(rasterTransfo, resAnalysis, plim, newRasters, cfgPath, cfgFlags):
 
     ax2.set_title('Peak Pressure distribution along the path between runs')
     ax2.legend(loc=4)
-    ax2.set_ylabel('l [m]')
+    ax2.set_ylabel('s [m]')
     ax2.set_ylim([s.min(), s.max()])
     ax2.set_xlim(auto=True)
     ax2.set_xlabel('$P_{max}(s)$ [kPa]')
 
-    outFileName = '_'.join([projectName, dirName, 'plim',
-                            str(int(plim)), 'slComparison'])
+    #outFileName = '_'.join([projectName, dirName, 'plim',
+    #                        str(int(plim)), 'slComparisonStat'])
+    outFileName = '_'.join([projectName, 'plim',  str(int(plim)), 'slComparisonStat'])
+
+
+    pU.saveAndOrPlot(cfgPath, cfgFlags, outFileName, fig)
+
+
+def visuMass(resAnalysis, cfgPath, cfgFlags):
+    """
+    Plot and save the comparison between current simulation and Reference
+    in the run-out area
+    """
+    ####################################
+    # Get input data
+    # read paths
+    projectName = cfgPath['projectName']
+    #dirName = cfgPath['dirName']
+    # read data
+    entMassArray = resAnalysis['entMassArray']
+    totalMassArray = resAnalysis['totalMassArray']
+    time = resAnalysis['time']
+
+    ############################################
+    # prepare for plot
+    Title = ['Entrained ', 'Total ']
+    Unit = ['Entrained Mass ', 'Total Mass ']
+    DataMass = np.array(([None] * 2))
+    DataMass[0] = entMassArray
+    DataMass[1] = totalMassArray
```

Comment:
adjust variable naming: Title to title etc

Reply:
moved to issue